<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MostrarReservas</class>
 <widget class="QMainWindow" name="MostrarReservas">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>659</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mostrar Reservas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0;
color: #333333;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #2D2D2D;
color: #E0E0E0;
</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>572</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>492</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">handle {
    background-color: #444444;
    border: 1px solid #555555;
}
</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="handleWidth">
     <number>21</number>
    </property>
    <widget class="QListWidget" name="asg_listaWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>119</width>
       <height>492</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3C3C3C;
border: 1px solid #555555;
border-radius: 5px;
color: #E0E0E0;

QListWidget::item {
    padding: 5px;
}

QListWidget::item:selected {
    background-color: #505050;
    color: #FFFFFF;
}
</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="asg_tablaWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>190</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>498</width>
       <height>492</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3C3C3C;
border: 1px solid #555555;
border-radius: 5px;
gridline-color: #555555;
color: #E0E0E0;

QHeaderView::section {
    background-color: #444444;
    color: #FFFFFF;
    font-weight: bold;
    border: 1px solid #666666;
}

QTableWidget::item {
    border: 1px solid #555555;
}

QTableWidget::item:selected {
    background-color: #505050;
    color: #FFFFFF;
}
</string>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideLeft</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::PenStyle::SolidLine</enum>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>122</number>
     </attribute>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Persona</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tel√©fono</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de Reserva</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="asg_botonReservar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Acceder a la pantalla de Reservas</string>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #666666;
}

</string>
    </property>
    <property name="text">
     <string>Reservar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asg_edit_reserva">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Acceder a la pantalla de Reservas para editar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #666666;
}</string>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
